apps/keyboard/style/keyboard.css

```diff
@@ -1,4 +1,10 @@
 html {
+  /**
+   * font-size is to be overwritten by JavaScript here.
+   * We are absuing CSS unit "rem" as a grid to describe the size of our elements.
```

Comment:
typo: absuing / abusing.

I don't think it's abuse to use rem like this, but soit.